### PR TITLE
General cleanup for surfacing docs on RTD

### PR DESCRIPTION
**Pull request by asuresh4**
Added missing images, normalized installation steps, text in metrics section, escaped underscores, asterisk, use same ("\`\`\` \`\`\`" ) pattern for code blocks, removed dead links

### CODE REVIEW

File: lib-python/README.md [hunk 1]
```diff
@@ -239,22 +196,13 @@ gauge('test').set_value(42)
...
-that exit right after calling the send method (as
-can be the case if your script is sending just one
-datapoint) may not yield a datapoint in a chart on
-the SignalFx service.
+The Python client library is mainly targeted towards sending a continuous stream of metrics, and was implemented to be asynchronous. As a result, short-lived scripts that exit right after calling the send method (as can be the case if your script is sending just one datapoint) may not yield a datapoint in a chart on the SignalFx service.
 
-To work around this problem, register an `atexit`
-function to cleaning stop the datapoint sending thread
-when your program exits:
+To work around this problem, register an `atexit` function to cleaning stop the datapoint sending thread when your program exits:
```
Comment (charliesignalfx) [inline lib-python/README.md]: This sentence was never phrased well.
Please rephrase it to:
"To work around this problem, register an `atexit` function to cleanly stop the datapoint sending thread when your program exits:"

**Comment by barbara-sfx** [comment]
Please do not merge until Akash and I have confirmed our docs push happened and is successful.

**Comment by asuresh4** [comment]
@barbara-sfx I have removed all the possible cross references. Now, the docs site would refer to the github integrations repo and vice-versa is avoided in all possible cases, making this PR independent of the docs site.
